src/v/storage/log_manager.h

```diff
@@ -205,7 +205,7 @@ class log_manager {
 private:
     using logs_type
       = absl::flat_hash_map<model::ntp, std::unique_ptr<log_housekeeping_meta>>;
-    using compaction_list_type
+    using housekeeping_list_type
```

Comment:
i was going to rely on this list, but in the end i didn't, so this name change could be reverted back